news/2020-02-24-mappers-report-february.md

```diff
@@ -0,0 +1,134 @@
+- **osu!taiko:** [\_DUSK\_](https://osu.ppy.sh/users/6092181), [Capu](https://osu.ppy.sh/users/2474015), [Gamelan4](https://osu.ppy.sh/users/9856910), [Hoshikuz](https://osu.ppy.sh/users/9892196), [incandescence](https://osu.ppy.sh/users/6256027), [kanpakyin](https://osu.ppy.sh/users/394326), [Stefan](https://osu.ppy.sh/users/626907), [Vulkin](https://osu.ppy.sh/users/4901066) & [Yales](https://osu.ppy.sh/users/2377881)
+- **osu!catch:** [Lacrima](https://osu.ppy.sh/users/4915649) & [Dapuluous](https://osu.ppy.sh/users/8140944)
+- **osu!mania:** [Voxa](https://osu.ppy.sh/users/9229539), [Dubstek](https://osu.ppy.sh/users/9555243), [UnluckyCroco](https://osu.ppy.sh/users/9461160) & [Amiichii](https://osu.ppy.sh/users/9228569)
+
+We'd like to use this section as a reminder that you can apply as a Beatmap Nominator at any time over at [BN/NAT website](https://bn.mappersguild.com/) as long as you reach the activity requirements. We hope to see some of your names on this list next month!
+
+### BN Retirements
+
+Next, we have those who have taken their leave from the group. We'd like to thank them for their tremendous work and effort within the BNs and wish them the best with their newly obtained freedom!
+
+- **osu!:** [Trynna](https://osu.ppy.sh/users/2652951)
+- **osu!taiko:** [qoot8123](https://osu.ppy.sh/users/766371)
+- **osu!catch:** [Sanyi](https://osu.ppy.sh/users/7496029), [Du5t](https://osu.ppy.sh/users/6053071) & [Rocma](https://osu.ppy.sh/users/566276)
+- **osu!mania:** [Tailsdk](https://osu.ppy.sh/users/6751666), [Critical_Star](https://osu.ppy.sh/users/3793196), [Kyousukee](https://osu.ppy.sh/users/8842107) & [MEGAtive](https://osu.ppy.sh/users/3094101)
+
+### NAT Additions
+
+In addition to the recently added BNs, some new NAT members have been brought on!
+
+The first addition we got on the 6h January was **[yaspo](https://osu.ppy.sh/users/4945926)** who served as a Beatmap Nominator for roughly over a year, he will now support the NAT for the osu! game mode. He will mainly help with BN evaluations and applications. He has also shown his skills in leading the NAT meetings already! Welcome aboard, yaspo!
+
+Our second addition we got on the 26th January is **[Greaper](https://osu.ppy.sh/users/2369776)**! With being previously a Beatmap Nominator for almost 4 months, he will now support the NAT for the osu!catch game mode. He will mainly focus on doing BN evaluations and applications and make sure, the Beatmap Nominator group will grow properly for osu!catch! So welcome aboard Greaper!
+
+## <a id="mapping-contests"></a>Mapping Contests
+
+If you are organizing a mapping contest that will begin within the next 1-3 months, feel free to submit it through [this form](https://docs.google.com/forms/d/e/1FAIpQLSeVPi2KkcWiEsJzsEs1ir79HrP2jE_Sy2DCX_4nR_EECFTX4Q/viewform?usp=sf_link) for a chance of being featured on next month's installment of the Mappers' Report. We are looking to highlight more mapping contests from the community!
```

Comment:
can't you just look at the forums when you write the post? there aren't too many mapping contests anyway. someone hosting a new contest probably won't go back submit it to this form

Reply:
maybe we could also double check, but currently mapping contest threads get spread throughout language specific forums, mode specific forums, and beatmap projects forum, so constantly checking in all of these locations (and potentially others if contests are held elsewhere) is not feasible to always catch everything. so a form would be a good back up in this case if the contest organizer remembers it.

Reply:
they're not likely to remember tbh, you're better off actively looking for these things

Reply:
Hmm.. couldn't we do both? I mean sure we can keep an eye on the forums but as Noffy said it's kinda difficulty to find all of them because sometimes they just get posted in language specific forums, sometimes in the tournament forum.

Just an idea: Could we make a thread in the Tournament Forum with the form which we also use in the news post? So the people who want to have a little advertising of their contest don't have to wait/look for a news post and the form would be available all the time.

Reply:
that sounds like a good compromise, yes

Reply:
Alright, i created a thread. I will move it to the tournament forums (and pin it there) once the news post goes live!

Reply:
not going to reference the thread in the post?

Reply:
Replaced the direct form link with the forum thread. Don't think we need it twice linked in one sentence.